node/src/components/consensus/highway_core/active_validator.rs

```diff
@@ -77,12 +83,14 @@ impl<C: Context> ActiveValidator<C> {
         timestamp: Timestamp,
         state: &State<C>,
     ) -> (Self, Vec<Effect<C>>) {
+        let next_round_exp = state.params().init_round_exp();
         let mut av = ActiveValidator {
             vidx,
             secret,
-            next_round_exp: state.params().init_round_exp(),
+            next_round_exp,
             next_timer: Timestamp::zero(),
             next_proposal: None,
+            round_success_meter: RoundSuccessMeter::new(next_round_exp, timestamp),
```

Comment:
That's the current timestamp, is it?
What if we start our node some time before genesis (as we are expected to): Will that cause all the "rounds" before genesis, where nobody creates any votes, to be recorded as "failed"?